.github/workflows/aksel-dev.yml

```diff
@@ -34,6 +34,9 @@ jobs:
       - name: Install dependencies
         run: yarn
 
+      - name: Create empty .env file (Can be removed when env-file-if-exists is implemented in node v22)
```

Comment:
Right now when using `--env-file`, the script breaks when there is no "physical" .env-file located in the dir. This will be [fixed in node v22](https://nodejs.org/api/cli.html#--env-file-if-existsconfig) when we can use `--env-file-if-existsconfig`, but for now this fixes it by adding an empty file. 

The scripts needing the env-variables throws if the tokens does not exist, meaning the github-action way of handling env works as expected by making sure no actual env-file is needed.
```
- name: Build application
        env:
            ...
```

Reply:
Annoying that they don't give a date for when v22 becomes LTS, just "October" 😅